index.html

```diff
@@ -80,7 +80,7 @@ <h1 id="title"><i class="todo">PROPOSED</i> Decentralized Identifier Working Gro
 <a href="https://www.w3.org/2019/did-wg/">Decentralized Identifier Working Group</a>
 is to maintain the <a href="https://www.w3.org/TR/did-core/">Decentralized Identifiers (DIDs)</a>
 specification and related Working Group Notes. The WG will also seek consensus
-around the specification of DID Methods.
+around the specification of DID Methods and DID Resolution.
```

Comment:
```suggestion
around the specification of DID Methods and DID Resolution (incl. DID URL Dereferencing).
```

Reply:
Best not to use abbreviations like `incl.`  Please change all three to `including`, or consider defining `DID Resolution` to include `DID URL Dereferencing` up front (which would make the subsequent parenthetical `including` notes redundant and unnecessary).

Reply:
addressed in [409f1c6](https://github.com/w3c/did-wg-charter/pull/25/commits/409f1c61b8000805fe33cb67e8e8ee43ad8bf25a)